### PR TITLE
Multiple fixes

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

**Comment by johngian** [comment]
@akatsoulas r?

File: iam_profile_faker/cli.py [hunk 1]
```diff
@@ -43,15 +44,19 @@ def create_batch(count):
...
-    db = Flata(os.path.join(path, dbname), storage=JSONStorage)
-    table = db.table('users')
 
+    click.echo('Creating database {0}'.format(dbname))
+
+    db = TinyDB(os.path.join(path, dbname))
     users = V2ProfileFactory().create_batch(count, export_json=False)
-    table.insert_multiple(users)
+    for user in users:
+        db.insert(user)
```
Comment (akatsoulas) [inline iam_profile_faker/cli.py]: nit: you can also use insert_multiple

File: iam_profile_faker/v2_api.py [hunk 2]
```diff
@@ -40,27 +39,28 @@ class PersistentUsers(Resource):
...
-        db_name = _load_db()
-        table = Flata(db_name, storage=JSONStorage).table('users')
-        query = table.search(Query().id == int(user_id))
-        return query
+        db = TinyDB(_load_db())
+        return db.all()[user_id]
 
 
-api.add_resource(RandomUsers, '/', '/users')
+api.add_resource(RandomUsers, '/users')
```
Comment (akatsoulas) [inline iam_profile_faker/v2_api.py]: Let's keep `/` too

**Review by akatsoulas — APPROVED** [review]
r+wc

**Pull request merged**